float64 lastScanTime
int64 nScanPts
float32[] scanPts
float32 angStep
float32 minAngle
float32 maxAngle
float64 lastOdomTime
geometry_msgs/Pose pose
